app/controllers/routes_controller.rb

```diff
@@ -63,11 +63,18 @@ def destroy
       format.json { head :no_content }
     end
   end
+  def yours
+    @route = Route.where(user_ID: current_user.id)
+  end
```

Comment:
does this work? Don't need to change anything - just a question..

Reply:
as far as I'm aware it is currently working 💃